src/controller/tests/data_model/TestWrite.cpp

```diff
@@ -43,15 +44,20 @@ constexpr EndpointId kTestEndpointId       = 1;
 constexpr DataVersion kRejectedDataVersion = 1;
 constexpr DataVersion kAcceptedDataVersion = 5;
 
+constexpr uint8_t kExampleClusterSpecificSuccess = 11u;
+constexpr uint8_t kExampleClusterSpecificFailure = 12u;
+
 enum ResponseDirective
 {
     kSendAttributeSuccess,
     kSendAttributeError,
     kSendMultipleSuccess,
     kSendMultipleErrors,
+    kSendClusterSpecificSuccess,
+    kSendClusterSpecificFailure,
 };
 
-ResponseDirective responseDirective;
+ResponseDirective gResponseDirective;
```

Comment:
This global is not really reset so I wonder if this makes tests have sideffects.

Could we initialize it to something explicitly (guessing it gets success now because BSS is 0 by default) and use https://github.com/project-chip/connectedhomeip/blob/master/src/lib/support/Scoped.h#L32 ?

I also realize this is not the scope of your PR really... so if you feel this is better later/separate that is fine as well.

Reply:
:)

Reply:
Done